server/src/utils.ts

```diff
@@ -186,16 +186,20 @@ export const toCamelCase = (text: string): string => {
     .replace(/(\s|-)+/g, "");
 };
 
+const readBsConfig = (projDir: p.DocumentUri) => {
+  let bsconfigFile = fs.readFileSync(
+    path.join(projDir, c.bsconfigPartialPath),
+    { encoding: "utf-8" }
+  );
+
+  return JSON.parse(bsconfigFile);
```

Comment:
How about catching the parse error inside this function. So there's no need to wrap the callers in a try block. Assuming that was the only reason to use a try block in the caller, which I think is true at least for the previous uses of bsconfig parse.